tritium-metrics/build.gradle

```diff
@@ -26,6 +26,8 @@ dependencies {
     testImplementation 'junit:junit'
     testImplementation 'org.assertj:assertj-core'
     testImplementation 'org.mockito:mockito-core'
+    testImplementation 'io.undertow:undertow-core'
+    testImplementation 'com.squareup.okhttp3:okhttp'
```

Comment:
Are we okay with bringing these in for testing?

Reply:
I'm fine with these as test dependencies